SPECS/golang/golang.spec

```diff
@@ -56,19 +60,44 @@ popd
 mv -v %{_topdir}/BUILD/go-bootstrap %{_libdir}/golang
 export GOROOT=%{_libdir}/golang
 
-# Use go1.4 bootstrap to compile go%{bootstrap_compiler_version} (bootstrap)
+# Use go1.4 bootstrap to compile go%{bootstrap_compiler_version_0}
 export GOROOT_BOOTSTRAP=%{_libdir}/golang
-mkdir -p %{_topdir}/BUILD/go%{bootstrap_compiler_version}
-tar xf %{SOURCE2} -C %{_topdir}/BUILD/go%{bootstrap_compiler_version} --strip-components=1
-pushd %{_topdir}/BUILD/go%{bootstrap_compiler_version}/src
+mkdir -p %{_topdir}/BUILD/go%{bootstrap_compiler_version_0}
```

Comment:
Please now also update the build comment at the very beginning of the `%build` stage to correctly reflect current dependencies between the built and bootstrap versions.